<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResolutionInstance</class>
 <widget class="QDialog" name="ResolutionInstance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Résolution d'instances</string>
   </property>
  </widget>
  <widget class="QLabel" name="fichierInstanceLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fichier d'instance :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="choisirFichierLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>421</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="choisirFichierPushButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>130</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Choisir un fichier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="validerPushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>390</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Résoudre</string>
   </property>
  </widget>
  <widget class="QPushButton" name="annulerPushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="fichierRadioButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Fichier d'instance</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="dossierRadioButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Dossier d'instances</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>531</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="autoScrollMargin">
    <number>16</number>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <attribute name="headerVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
   <item>
    <property name="text">
     <string>Resolution exacte indexée temps</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Resolution exacte indexée jobs</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CCmax basé sur la somme des ressources de chaque job</string>
    </property>
    <item>
     <property name="text">
      <string>Affectation priorisant la machine avec le moins de charge</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Affectation machine par machine</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>CCmax basé sur la ressource maximale de chaque job</string>
    </property>
    <item>
     <property name="text">
      <string>Affectation priorisant la machine avec le moins de charge</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Affectation machine par machine</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Somme des ressources de chaque job</string>
    </property>
    <item>
     <property name="text">
      <string>Affectation priorisant la machine avec le moins de charge</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Affectation machine par machine</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Valeur moyenne des ressources des sous-ensembles maximaux</string>
    </property>
    <item>
     <property name="text">
      <string>Affectation priorisant la machine avec le moins de charge</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Affectation machine par machine</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
